Remove some well-known directories from the LD_RUN_PATH

--- lib/ExtUtils/Liblist/Kid.pm.pix	2002-06-01 19:03:16.000000000 +0200
+++ lib/ExtUtils/Liblist/Kid.pm	2002-06-23 21:39:37.000000000 +0200
@@ -46,6 +46,7 @@
     my(@searchpath); # from "-L/path" entries in $potential_libs
     my(@libpath) = split " ", $Config{'libpth'};
     my(@ldloadlibs, @bsloadlibs, @extralibs, @ld_run_path, %ld_run_path_seen);
+    $ld_run_path_seen{$_} = 1 foreach qw(/usr/X11R6/lib /usr/lib /lib);
     my(@libs, %libs_seen);
     my($fullname, $thislib, $thispth, @fullname);
     my($pwd) = cwd(); # from Cwd.pm
